apps/docs/content/guides/database/replication/etl.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: ETL
+description: Replicate your Supabase database to external destinations
+---
+
+`pg_replicate` simplifies the process of syncing your data to an external destination. This could be another database, a data warehouse, or a specialized service such as customer management or business analytics software.
+
+<Admonition type="caution">
+
+`pg_replicate` is currently in alpha.
+
+</Admonition>
+
+Syncing requires a source and a destination. The source is your Supabase database, defined in the [**Publications**](https://supabase.com/dashboard/project/_/database/publications) section of the dashboard.
+
+The destination is a third party location, defined in the [**Replication**](https://supabase.com/dashboard/project/_/database/replication) section of the dashboard. See the [supported destinations](#destinations).
+
+## Sources
+
+Choose what data to sync by creating a publication in your database. A publication is a set of changes, filtered by table and change type.
+
+### Managing publications
+
+1. Go to the [Database](https://supabase.com/dashboard/project/_/database/tables) page in the Dashboard.
+2. Click on **Publications** in the sidebar.
+3. Control which database events to send by toggling **Insert**, **Update**, and **Delete**.
```

Comment:
We should probably omit this step because replication in our case will work only when all `insert`, `update` and `delete` events are propagated, even though Postgres allows users to pick them.